lib/members_page.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'scraped'
+
+# This class represents a page listing members of the given legislature
+class MembersPage < Scraped::HTML
+  decorator Scraped::Response::Decorator::CleanUrls
+
+  field :member_urls do
+    noko.css('.bio-member-detail-1 a/@href').map(&:to_s)
```

Comment:
We usually use `.text` rather than `.to_s` there, so unless there's a reason why that doesn't work here, it's better to be consistent.